### PR TITLE
Fix Docker

### PR DESCRIPTION
**Pull request by WhiteZh**
Docker setup was broken previously. The reason was that some latest versions of dependencies no longer support the Ruby (v2.6.3) the project is currently using. Solved by using earlier versions of them which do support Ruby v2.6.3.

### CODE REVIEW

**Review by murtazazarkhwan — APPROVED** [review]
Thanks @WhiteZh, LGTM

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <ul class="list-inline mx-auto justify-content-center">
     <li>
```
Comment (murtazazarkhwan) [inline app/views/shared/_navbar.html.erb]: @WhiteZh this commit has a different purpose than the PR itself, it would be best if you can please submit a separate PR for this Navbar fix.

**Review by murtazazarkhwan — CHANGES_REQUESTED** [review]

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (WhiteZh) [reply app/views/shared/_navbar.html.erb]: Understood. I have created a new pull request. Sorry, I am not very familiar with pull request system, it was a mistake.

Comment (WhiteZh) [reply app/views/shared/_navbar.html.erb]: Also, I have moved this branch back to the dockerfile fixing commit. So the unifying heights commit is erased from this branch.

Comment (murtazazarkhwan) [reply app/views/shared/_navbar.html.erb]: Thank you so much
